tests/test_zarr_compat.py

```diff
@@ -14,17 +14,16 @@ def airtemp_ds():
     return ds
 
 
-@pytest.mark.xfail(reason='We are not populating the default compressors/filters yet.')
 def test_zmetadata_identical(airtemp_ds):
     zarr_dict = {}
+    airtemp_ds = airtemp_ds.chunk(dict(airtemp_ds.dims))
```

Comment:
I am using **one chunk** here until we figure out how zarr determines chunk sizes automatically.